python/reference-apps/topics/main.py

```diff
@@ -0,0 +1,5 @@
+def topic():
```

Comment:
could it be a variable instead of a function? I think having a function that just returns a value is a bit confusing.